docs/_writing-tests/index.md

```diff
@@ -11,43 +11,47 @@ There's also a load of [general guidelines][] that apply to all tests.
 
 ## Test Type
 
-There are four main test types:
+Tests in this project use a few different approaches to verify expected
+behavior. The tests can be classified based on the way they express
+expectations:
 
-* [Reftests][] should be used to test rendering and layout. They
-  consist of two or more pages with assertions as to whether they
-  render identically or not.
+* Rendering tests should be used to verify that the browser graphically
+  displays pages as expected. See the [rendering test guidelines][rendering]
+  for tips on how to write great rendering tests. There are a few different
+  ways to write rendering tests:
 
-* [testharness.js][] tests should be used (where
-  possible!) for testing everything else. They are built with the
-  testharness.js unit testing framework, and consist of assertions
-  written in JavaScript.
+  * [Reftests][] should be used to test rendering and layout. They consist of
```

Comment:
Note that this is increasingly not the case; more and more of our layout tests are testharness.js tests. Changing the documentation may be more contentious however.

Reply:
More contentious, and maybe more involved as well. We might want to make some different/more specific recommendations for how authors should choose between the types. Let's discuss that in a follow-up.

Reply:
@jugglinmike can you file an issue for this?

Reply:
Sure: gh-16602